docs/examples/control_ish_ufschem-example.yaml

```diff
@@ -0,0 +1,294 @@
+    # ^ List of domain names. If domain_type = all,
+    #   the domain name is used in the plot title.
+
+      #   #### View unique state and country values 
+  #   # an.paired['ish_ufschem_v1']
+  #   # df = an.paired['ish_ufschem_v1'].obj.to_dataframe()
+  #   # df.columns
+  #   # df["country"].unique()
+  #   # df["state"].unique()
+
+    data: ["ish_ufschem_v1"]
+    # ^ make this a list of pairs in obs_model
+    #   where the obs is the obs label and model is the model_label
+    data_proc: # optional
+      rem_obs_nan: True
+      # ^ True: Remove all points where model or obs variable is NaN.
+      #   False: Remove only points where model variable is NaN.
+      ts_select_time: "time_local"  # `ts_` indicates this is time series plot-specific
+      # ^ Time used for avg and plotting
+      #   Options: 'time' for UTC or 'time_local'
+      ts_avg_window: "h"
+      # ^ Options: None for no averaging, pandas resample rule (e.g., 'h', 'D')
+      set_axis: True
+      # ^ If true, add `vmin_plot` and `vmax_plot` for each variable in obs.
+
+  plot_grp2:
+    type: "spatial_bias"
+    fig_kwargs: # optional; For all spatial plots, specify map_kwargs here too.
+      states: True  # such as whether to show the state boundaries
+      figsize: [10, 5]
+    text_kwargs:
+      fontsize: 16.
+    domain_type: ["all", "state"]
+    domain_name: ["CONUS", "CA"]
+    data: ["ish_ufschem_v1"]
+    data_proc:
+      rem_obs_nan: True
+      set_axis: True
+
+  plot_grp3:
+    type: "spatial_overlay"
+    fig_kwargs:
+      states: True
+      figsize: [10, 5]
+    text_kwargs:
+      fontsize: 16. 
+    domain_type: ["all", "all"]
+    domain_name: ["CONUS", "GLOBAL"]
+    
+    # to get CONUS, you need to do "all" "CONUS"
+
+    data: ["ish_ufschem_v1"]
+    data_proc:
+      rem_obs_nan: True
+      set_axis: True
+      
+  plot_grp4:
+    type: "boxplot"
+    fig_kwargs:
+      figsize: [8, 6]
+    text_kwargs:
+      fontsize: 20.
+    #gridlines: True
+    domain_type: ["all"]
+    domain_name: ["GLOBAL"]
+    data: ["ish_ufschem_v1"]
+    data_proc:
+      rem_obs_nan: True
+      set_axis: False
+
+  plot_grp5:
+    type: "multi_boxplot"
+    fig_kwargs:
+      figsize: [10, 8]
+    text_kwargs:
+      fontsize: 20.
+    gridlines: True
+    domain_type: ["all"]
+    domain_name: ["GLOBAL"]
+    interval_list: [273.15, 281.15, 289.15, 297.15, 305.15]
+    interval_labels: ["[273.15, 281.15)", "[281.15, 289.15)", "[289.15, 297.15)", "[297.15, 305.15)"]
+    interval_var: t
+
+    # to create an interval boxplot, simply specify the intervals you would like to plot AND the variable name. In this case, wind speed is being plotted at several intervals.
+
+    model_name_list: ['ISH', "ufschem_v1"]
+    data: ["ish_ufschem_v1"]
+    data_proc:
+      rem_obs_nan: True
+      set_axis: False
+      
+  plot_grp6:
+    type: "rose_plot"
+    text_kwargs:
+      fontsize: 16.
+    domain_type: ["all", "state"]
+    domain_name: ["CONUS", "CA"]
```

Comment:
Need to change CONUS here to GLOBAL. But it is maybe less useful to make a rose plot with GLOBAL data. So you may just want to use EPA region R9 to showcase these differences instead.